sgkit_plink/tests/data/plink_sim_10s_100v_10pmiss.bim

```diff
@@ -0,0 +1,100 @@
+1	1:1:A:C	0.0	1	C	A
```

Comment:
Can you document somewhere how these files were generated.

Reply:
The notebooks are in [eric-czech/sgkit-dev/notebook/plink](https://github.com/eric-czech/sgkit-dev/tree/master/notebook/plink) but I was going to wait on https://github.com/pystatgen/sgkit/issues/15 to figure out where to put it.  I like the idea of this being in a separate repo because setting up hail isn't something I think we want to do in this project so it feels odd to put code here that uses it.  

What do you think?  Perhaps it should be a standard to at least put at a README in those data folders that contain links out to the code that generated it.